master/getting-started/kubernetes/installation/aws.md

```diff
@@ -2,7 +2,7 @@
 title: Deploying Calico and Kubernetes on AWS
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on AWS using the [Calico CNI plugin][calico-cni]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+These instructions allow you to set up a Kubernetes cluster with Calico networking on AWS using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
```

Comment:
where does this hyperlink redirect to? I'm not familiar with this `[][]` format, but our docs use `[](url)` format. More curious than anything